nautobot/ipam/graphql/types.py

```diff
@@ -9,22 +9,13 @@ class IPAddressType(gql_optimizer.OptimizedDjangoObjectType):
     """Graphql Type Object for IPAddress model."""
 
     address = graphene.String()
-    interface = graphene.Field("nautobot.dcim.graphql.types.InterfaceType")
-    vminterface = graphene.Field("nautobot.virtualization.graphql.types.VMInterfaceType")
+    ip_version = graphene.Int()
```

Comment:
I'm actually surprised this needs to be added. I was pretty sure this would come through from the model just fine.

Reply:
I was surprised too. All I can figure is that graphene takes the fact that the field has `choices` as an indicator that it should be treated as an enum type instead of an integer type.